dockerfiles/theia-dev/docker/ubi8/install-dependencies.dockerfile

```diff
@@ -1,3 +1,5 @@
 USER root
 RUN yum install -y curl make cmake gcc gcc-c++ python2 git openssh less bash tar gzip \
-    && yum -y clean all && rm -rf /var/cache/yum
+    && yum -y clean all && rm -rf /var/cache/yum && \
+    ln -s /usr/bin/python2.7 /usr/bin/python; python --version && \
```

Comment:
could you please move that to https://github.com/redhat-developer/codeready-workspaces-theia

Reply:
I don't see why this can't be in upstream. It doesn't prevent upstream from working, and it's not like the upstream image is even being built as part of CI.